hack/release-tar

```diff
@@ -22,7 +22,24 @@ if [ -n "$PLATFORMS" ]; then
   platformFlag="--platform=$PLATFORMS"
 fi
 
-buildxCmd build $platformFlag $cacheFromFlags \
+output=$(mktemp -d -t buildkit-output.XXXXXXXXXX)
+
+buildxCmd build $platformFlag $cacheFromFlags $(buildAttestFlags) \
+  --build-arg "BUILDKIT_MULTI_PLATFORM=true" \
   --target release \
-  --output "type=local,dest=$OUT" \
+  --output "type=local,dest=$output" \
   $currentcontext
+
+for pdir in "${output}"/*/; do
+  (
+    cd "$pdir"
+    releasetar=$(find . -name '*.tar.gz')
+    filename=$(basename "${releasetar%.tar.gz}")
+    find . -name '*.json' -print0 | tar -czf "${filename}.attest.tar.gz" --null --files-from -
```

Comment:
could we keep provenance separate and only create `sbom.tar.gz` for the rest if we can't get a single sbom file atm.

Reply:
```
$ tree release-out/
release-out/
├── buildkit-v0.11.0-rc2-2-g28a138369.linux-amd64.provenance.json
├── buildkit-v0.11.0-rc2-2-g28a138369.linux-amd64.sbom.tar.gz
└── buildkit-v0.11.0-rc2-2-g28a138369.linux-amd64.tar.gz
```